packages/generator/templates/app/src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
```

Comment:
Same as above: We should make this a server component, add the demo-metadata export and move the hooks inside the `LoginForm` client component.